jobs/integr8ly/nightly-pipeline-qe-pony.yaml

```diff
@@ -0,0 +1,193 @@
+---
+
+- job:
+    name: nightly-pipeline-qe-pony
+    project-type: pipeline
+    description: "Nightly pipeline for testing on POC (originally Pony) clusters."
+    sandbox: false
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: 20
+    parameters:
+        - string:
+            name: GH_CLIENT_ID
+            description: "GitHub OAuth Client ID, required for some of the walkthroughs. Can be left empty"
+        - string:
+            name: GH_CLIENT_SECRET
+            description: "GitHub OAuth Client Secret, required for some of the walkthroughs. Can be left empty"
+        - bool:
+            name: SELF_SIGNED_CERTS
+            default: false
+            description: "Indicates whether the cluster uses self signed certificates or not.
+This needs to be changed to 'false' for Integreatly Workshop since it uses valid certificates."
+        - string:
+            name: RECIPIENTS
+            default: integreatly-qe@redhat.com
+            description: "Whitespace- or comma-separated list of recipient addresses"
+        - string:
+            name: INSTALLATION_REPOSITORY
+            default: https://github.com/integr8ly/installation.git
+            description: "Repository of the Integreatly installer"
+        - string:
+            name: INSTALLATION_BRANCH
+            default: master
+            description: "Branch of the installer repository"
+        - string:
+            name: TEST_SUITES_REPOSITORY
+            default: https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git
+            description: "Repository of the Integreatly test suites"
+        - string:
+            name: TEST_SUITES_BRANCH
+            default: master
+            description: "Branch of a repository of Integreatly test suites"
+        - string:
+            name: CLUSTER_URL
+            description: "URL of cluster for installation."
+        - string:
+            name: MASTER_URLS
+            description: "Comma separated list of URLs for master nodes of target cluster to be used in Ansible inventory file"
+        - string:
+            name: ANSIBLE_USER
+            default: root
+            description: "User for Ansible to access the master node of target cluster"
+        - string:
+            name: OC_USER
+            default: trepel
+            description: "Default OpenShift admin user when Integr8ly is not installed."
+        - string:
+            name: OC_PASSWORD
+            description: "Default OpenShift admin password when Integr8ly is not installed."    
+    dsl: |
+        try {
+            timeout(180) { ansiColor('gnome-terminal') { timestamps {
+                node('cirhos_rhel7') {        
+                    stage('Verify input') {
+                        if (!MASTER_URLS) {
+                            throw new hudson.AbortException('MASTER_URLS parameter is required!')
```

Comment:
Here only presence of one parameter is required, but there are more of them, which are required. I would either leave it out, or check if all required parameters are provided

Reply:
I added some

Reply:
I added some